### PR TITLE
Get Host Forensics - Generic - Added investigation and forensics outputs

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
Related to: https://jira-dc.paloaltonetworks.com/browse/CIAC-8220

## Description
Added outputs about forensics and investigation to the main playbook.

### CODE REVIEW

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_14.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Get host forensics - Generic
+
+- Added investigation data and forensics to the outputs of the playbook.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_14.md:6]:
```suggestion
Added investigation data and forensics to the outputs of the playbook.
```

**Comment by ShirleyDenkberg** [comment]
@michalgold @AdiPeret Doc review completed.

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_host_forensics_-_Generic.yml [hunk 2]
```diff
@@ -261,8 +263,36 @@ inputs:
   value: {}
   required: false
   description: Provide the machine IDs of the systems you want to retrieve.
-  playbookInputQuery: null
-outputs: []
+  playbookInputQuery:
+outputs:
+- contextPath: MicrosoftATP
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Get_host_forensics_-_Generic.yml:268]: are they any more integrations in the playbook? Besides Microsoft APT and illusive? That we might want to add their outputs as well?

Comment (idovandijk) [reply Packs/CommonPlaybooks/Playbooks/playbook-Get_host_forensics_-_Generic.yml:268]: Currently only these two :)

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**